pytmx/pytmx.py

```diff
@@ -1002,6 +1006,8 @@ def __init__(self, parent, node):
         self.y = 0
         self.width = 0
         self.height = 0
+        self.tilewidth = 0
```

Comment:
The addition of tilewidth and tileheight should be ignored. They ended up in the diff from another change I was working on, sorry.